starky/src/cross_table_lookup.rs

```diff
@@ -923,17 +866,10 @@ pub fn verify_cross_table_lookups<F: RichField + Extendable<D>, const D: usize,
     config: &StarkConfig,
 ) -> Result<()> {
     let mut ctl_zs_openings = ctl_zs_first.iter().map(|v| v.iter()).collect::<Vec<_>>();
-    for (
-        index,
-        CrossTableLookup {
-            looking_tables,
-            looked_table,
-        },
-    ) in cross_table_lookups.iter().enumerate()
-    {
+    for (index, CrossTableLookup { looking_tables }) in cross_table_lookups.iter().enumerate() {
         // Get elements looking into `looked_table` that are not associated to any STARK.
         let extra_sum_vec: &[F] = ctl_extra_looking_sums
```

Comment:
`ctl_extra_looking_sums` are now indexed the same as `cross_table_lookups`, not like looked tables.